src/net/yapbam/gui/util/FriendlyTable.java

```diff
@@ -114,62 +126,44 @@ protected void fillPopUp(JPopupMenu popup) {
 			}
 		}
 	}
-	

```

Comment:
Huum ... This method should not be moved to a subclass, even if it is not used today, the goal is to allow exporting any FriendlyTable by calling this method